### ISSUE TITLE
Dumpdata doesn't work

### ISSUE THREAD

**Issue by mlvn23** [ambitioninc/django-regex-field#11]
Seems like the field has an issue with serialization:

```
>>> python manage.py dumpdata allauth_extensions.blacklistedregexusernames --traceback

  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "D:\sw-dev\python\envs\django\lib\site-packages\django\core\management\__init__.py", line 399, in execute_from_command_line
    utility.execute()
  File "D:\sw-dev\python\envs\django\lib\site-packages\django\core\management\__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "D:\sw-dev\python\envs\django\lib\site-packages\django\core\management\base.py", line 242, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "D:\sw-dev\python\envs\django\lib\site-packages\django\core\management\base.py", line 285, in execute
    output = self.handle(*args, **options)
  File "D:\sw-dev\python\envs\django\lib\site-packages\django\core\management\commands\dumpdata.py", line 135, in handle
    use_natural_keys=use_natural_keys, stream=self.stdout)
  File "D:\sw-dev\python\envs\django\lib\site-packages\django\core\serializers\__init__.py", line 122, in serialize
    s.serialize(queryset, **options)
  File "D:\sw-dev\python\envs\django\lib\site-packages\django\core\serializers\base.py", line 50, in serialize
    self.handle_field(obj, field)
  File "D:\sw-dev\python\envs\django\lib\site-packages\django\core\serializers\python.py", line 51, in handle_field
    self._current[field.name] = field.value_to_string(obj)
  File "D:\sw-dev\python\envs\django\lib\site-packages\django\core\serializers\python.py", line 51, in handle_field
    self._current[field.name] = field.value_to_string(obj)
  File "D:\sw-dev\python\envs\django\lib\site-packages\regex_field\fields.py", line 49, in value_to_string
    return obj.pattern
AttributeError: 'BlackListedRegExUsernames' object has no attribute 'pattern'
```

**Comment by evansnj** [comment]
Any ideas on how to get around this? I'm having the same problem.

**Comment by wesokes** [comment]
@mlvn23 @evansnj this should be fixed in version 1.2.0

https://github.com/ambitioninc/django-regex-field/pull/30
